boards/FSM/Inc/App/App_FsmWorld.h

```diff
@@ -4,6 +4,7 @@
 #include "App_CanRx.h"
 #include "App_SharedHeartbeatMonitor.h"
 #include "App_FlowMeter.h"
+#include "App_WheelSpeedSensor.h"
```

Comment:
I would rename WheelSpeedSensor to just WheelSpeed for everything in this PR

Reply:
Done

Reply:
Actually thinking about it why don't we name it `App_WheelSpeedSensor`? We don't name `App_FlowMeter` `App_FlowRate`

Reply:
It's nicer to model things as real objects and calling it wheelspeedsensor doesn't convey more information than just wheel, it'd just add verbosity without clarity

Reply:
Actually I take this back, I was imagining that we'd add more interesting functionality such as `App_Wheel_ApplyTorque`, but due to the nature of our architecture FSM will only ever do the simple task of collecting sensor reading. I retract everything i said about changing `WheelSpeedSensor` to `Wheel`. Can you undo the changes?

Reply:
Okay changes undone